test/fixtures/vcr_cassettes/ra_club.yml

```diff
@@ -0,0 +1,56 @@
+    recorded_at: Tue, 08 Nov 2022 00:00:00 GMT
```

Comment:
Is that the right date? Like are these fixtures all based off one snap in time?

Reply:
yeah - i think F said that not having a frozen time in your test suite is asking for trouble - i think the `timecop` gem sets this iirc